modules/ossm-rn-new-features.adoc

```diff
@@ -43,6 +47,24 @@ This release of {SMProductName} addresses Common Vulnerabilities and Exposures (
 
 This release also adds the ability to disable the automatic creation of network policies.
 
+=== Component versions included in {ProductName} version 2.1.1
+
+|===
+|Component |Version
+
+|Istio
+|1.9.9
+
+|Envoy Proxy
+|1.17.1
+
+|Jaeger
+|1.24.1
```

Comment:
The latest Jaeger version is 1.30.1

Reply:
Updated the Jaeger version in the 2.1.2 section of rel notes.